layouts/index.html

```diff
@@ -96,7 +96,7 @@ <h1>Syntax</h1>
     {{ end -}}
         </li></ul>
     </nav>
-    {{- replace $spec "%%GRAMMAR%%" (transform.Highlight (partial "grammar.html" . ) "abnf") 1 | replaceRE "id=\"([^\"]+)\"" "id=\"spec-$1\"" | safeHTML -}}
+    {{- replace $spec "%%GRAMMAR.ABNF%%" (transform.Highlight (partial "grammar.html" . ) "abnf") 1 | replaceRE "id=\"([^\"]+)\"" "id=\"spec-$1\"" | safeHTML -}}
```

Comment:
@innovate-invent I’m not sure of this construct here. Does this reference the actual `GRAMMAR.ABNF` file?

Reply:
This is a manually implemented placeholder:
https://github.com/jmespath-community/jmespath.spec/blob/6db0c30449a13502da6904823036a33f515af8ec/GRAMMAR.ABNF#L48

It shouldn't be changed